NuGet.config

```diff
@@ -3,6 +3,7 @@
   <packageSources>
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-release/api/v3/index.json" />
     <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
+    <add key="msbuild" value="https://dotnet.myget.org/F/msbuild/api/v3/index.json" />
```

Comment:
Earlier work when it looked like I needed to pull specific msbuild packages. I don't think it's needed anymore.